easybuild/easyconfigs/w/WildMagic/WildMagic-5.17-GCCcore-10.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = "MakeCp"
+
+name = 'WildMagic'
+version = '5.17'
+
+homepage = "https://www.geometrictools.com/index.html"
+description = """Wild Magic 5.17"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://www.geometrictools.com/Downloads/']
+sources = ['WildMagic5p17.zip']
+checksums = ['b2caf7aa0c5309eb40965ed26439388dc1c93eb632312b40573f83e4746a9ecf']
+
+dependencies = [
+    ('binutils', '2.36.1'),
+    ('freeglut', '3.2.1'),
+]
+
+buildopts = "CFG=ReleaseDynamic -f makefile.wm5"
```

@stefan-wolfsheimer Please make sure that the right compiler is being used, this may explain why this easyconfig fails to build on some systems; see also #14594

```suggestion
buildopts = 'CFG=ReleaseDynamic -f makefile.wm5 CC="$CXX"'

---

I'll go ahead and apply this change, since it fixes a clear problem (`/usr/bin/g++` is used without specifying `CC`)